api/src/org/apache/cloudstack/api/command/admin/host/ListHostsCmd.java

```diff
@@ -206,7 +206,7 @@ public void execute() {
         } else {
             Pair<List<? extends Host>, Integer> result;
             Ternary<Pair<List<? extends Host>, Integer>, List<? extends Host>, Map<Host, Boolean>> hostsForMigration =
-                _mgr.listHostsForMigrationOfVM(getVirtualMachineId(), this.getStartIndex(), this.getPageSizeVal());
+                _mgr.listHostsForMigrationOfVM(getVirtualMachineId(), this.getStartIndex(), this.getPageSizeVal(), null);
```

Comment:
Why keyword is not used here? All API commands derived from BaseListCmd should have keyword.